migrations/20200730_02_create_table_transactions.py

```diff
@@ -0,0 +1,16 @@
+from yoyo import step
+
+__depends__ = {'20200730_01_create_table_users'}
```

Comment:
Why is it required?

Reply:
in this module you can create a dependency to undo migrations one at a time, if needed.
therefore, in each new file, the previous file is written.